deploy/default-azuredeploy.json

```diff
@@ -1,125 +1,129 @@
-                "reserved": true
-            }
-        },
-        {
-            "type": "Microsoft.Web/sites",
-            "apiVersion": "2018-11-01",
-            "name": "[variables('lowerCaseServiceName')]",
-            "location": "[variables('resourcesLocation')]",
-            "tags": {
-                "[variables('tagValue')]": "[variables('tagValue')]"
+            {
+              "name": "DOCKER_REGISTRY_SERVER_USERNAME",
+              "value": ""
+            },
+            {
+              "name": "DOCKER_REGISTRY_SERVER_PASSWORD",
+              "value": ""
             },
-            "dependsOn": [
-                "[concat('Microsoft.Web/serverfarms/', variables('serverfarmName'))]",
-                "[resourceId('Microsoft.Insights/components', variables('appInsightName'))]"
-            ],
-            "properties": {
-                "name": "[variables('lowerCaseServiceName')]",
-                "siteConfig": {
-                    "appSettings": [
-                        {
-                            "name":"CONVERSION_API_KEYS",
-                            "value":"[parameters('apiKey')]"
-                        },
-                        {
-                            "name":"APPINSIGHTS_INSTRUMENTATIONKEY",
-                            "value":"[reference(resourceId('Microsoft.Insights/components', variables('appInsightName')), '2014-04-01').InstrumentationKey]"
-                        },
-                        {
-                            "name": "DOCKER_REGISTRY_SERVER_URL",
-                            "value": "[concat('https://', variables('registryName'))]"
-                        },
-                        {
-                            "name": "DOCKER_REGISTRY_SERVER_USERNAME",
-                            "value": ""
-                        },
-                        {
-                            "name": "DOCKER_REGISTRY_SERVER_PASSWORD",
-                            "value": ""
-                        },
-                        {
-                            "name": "WEBSITES_ENABLE_APP_SERVICE_STORAGE",
-                            "value": "true"
-                        }
-                    ],
-                    "linuxFxVersion": "[concat('DOCKER|', variables('registryName'), '/fhirconverter:latest')]",
-                    "appCommandLine": "./deploy/webapp.sh",
-                    "alwaysOn": true
-                },
-                "serverFarmId": "[concat('/subscriptions/', subscription().subscriptionId, '/resourcegroups/', resourceGroup().name, '/providers/Microsoft.Web/serverfarms/', variables('serverfarmName'))]",
-                "clientAffinityEnabled": false,
-                "httpsOnly": true
+            {
+              "name": "WEBSITES_ENABLE_APP_SERVICE_STORAGE",
+              "value": "true"
             }
-        }
-    ]
+          ],
+          "linuxFxVersion": "[concat('DOCKER|', variables('registryName'), '/fhirconverter:', parameters('imageTag'))]",

```

Comment:
Do we need to list some recommended image tags in the readme doc?

Reply:
That is a good idea. I think also making some images with clear names would be a good idea. We can limit the valid input for this field, which will give the user a dropdown list of options instead of a free text box.

Reply:
Yes, we can modify this 'imageTag' parameter with some allowedValues.

Reply:
I added two versions for the moment: 2020-06-08 which was the last v1 image, and 2020-06-29 which is the most recent image at this time.